src/tk/mem.h

```diff
@@ -146,7 +155,7 @@ void mem_checkptr(void *ptr);
  *              return NULL
  */
 
-void *mem_malloc(size_t);
+void *mem_malloc(d_size_t);
 void *mem_calloc(size_t);
```

Comment:
With these translations, I do as little as possible for files other than the one being translated. The idea is to not mix up translations with refactorings, new features, bug fixes, etc. This is especially true since e2ir.d is a 6000 line file.

mem.h/.c has other problems when translating it to D, that'll be dealt with when it is its turn.